.travis.yml

```diff
@@ -8,9 +8,10 @@ jdk:
 install:
 
 - mvn clean install -B -V
+- mvn test
```

Comment:
It's unnecessary because maven will execute `test` when `install`.

Reply:
fixed~